### PR TITLE
Add Saxo Bank icon

### LINKED ISSUE #10546: Request: Saxo Bank
### Brand Name

Saxo Bank

### Website

https://home.saxo

### Popularity Metric

The Similarweb rank is 70,632. See https://www.similarweb.com/website/home.saxo

### Official Resources for Icon and Color

svg: <https://www.home.saxo/-/media/global/logos/saxo-2022/saxo-beinvested-logo-white.svg>

Logo taken from their homepage.

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by elliotwutingfeng**
image

**Issue:** closes #10546

**Similarweb rank:** 70,632

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

nil

### CODE REVIEW

File: _data/simple-icons.json
Comment (service-paradis) [inline _data/simple-icons.json:13226]: We should use the following links:
```suggestion
            "source": "https://www.home.saxo/about-us/media-center",
            "guidelines": "https://www.home.saxo/about-us/media-center"
```

File: _data/simple-icons.json
Comment (service-paradis) [inline _data/simple-icons.json:13225]: As mentioned in https://www.home.saxo/about-us/media-center, we should use `#051478`
```suggestion
            "hex": "051478",
```

File: icons/saxobank.svg
Comment (service-paradis) [inline icons/saxobank.svg:1]: By using `Logo/Digital use/Cropped/SAXO_be_invested__saxo_blue_logo_rgb.svg` from the Zip file found at https://www.home.saxo/about-us/media-center, I got the following differences (black being you version over the original in blue):
image

**Review by service-paradis — CHANGES_REQUESTED** [review]
Thank you @elliotwutingfeng for your contribution. I left a few comments. Can you take a look?

File: icons/saxobank.svg
Comment (elliotwutingfeng) [reply icons/saxobank.svg:1]: 730MB :scream:

File: icons/saxobank.svg
Comment (service-paradis) [inline icons/saxobank.svg:1]: Icons is matching perfectly now, thank you! There is a lot of extraneous nodes though in the "BE INVESTED" part. Mostly, the `B`, `S` and the `D`. It is not always easy to remove those kind of extra nodes. Can you give this a try and let me know?

**Review by service-paradis — CHANGES_REQUESTED** [review]

File: icons/saxobank.svg
Comment (elliotwutingfeng) [reply icons/saxobank.svg:1]: @service-paradis Will need help on this one

Comment (service-paradis) [reply icons/saxobank.svg:1]: Yeah, it is not easy. I just gave this a try and ends up with the following:
```svg
<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" viewBox="0 0 24 24"><path d="M2.508 6.791c-1.342.004-2.22.815-2.22 1.793 0 1.989 3.21 1.258 3.21 2.396 0 .4-.438.668-1.074.668-.67 0-1.104-.366-1.29-.93L0 11.083c.265.967 1.19 1.592 2.393 1.592 1.338 0 2.316-.748 2.316-1.766 0-2.061-3.232-1.39-3.232-2.426 0-.421.41-.67.959-.67.6 0 .912.288 1.076.612l1.119-.36c-.298-.796-1.12-1.273-2.123-1.273zm18.508.004c-1.677 0-2.979 1.323-2.979 2.941 0 1.618 1.302 2.942 2.979 2.942 1.676 0 2.984-1.324 2.984-2.942s-1.308-2.941-2.984-2.941zm-5.258.088L14.572 8.65l-1.185-1.763h-1.352l1.861 2.736-2.017 2.965h1.318l1.358-2.022 1.355 2.022h1.358L15.23 9.596l1.844-2.713zm-7.68.004-2.428 5.7h1.243l.496-1.196h2.388l.49 1.19.782-1.153L9.11 6.887Zm12.938.98c.996 0 1.752.782 1.752 1.87 0 1.086-.756 1.868-1.752 1.868-.997 0-1.75-.782-1.75-1.869s.768-1.869 1.75-1.869zm-12.44.608.785 1.927H7.795Zm6.291 6.843c-.307.003-.64.226-.625.541 0 .131.044.237.131.317.122.156.581.224.734.291.11.04.164.114.161.199 0 .211-.244.273-.366.262-.16.01-.378-.08-.381-.287h-.328c.02.45.423.57.713.568.349.005.7-.165.696-.543.005-.337-.323-.488-.54-.529l-.17-.045c-.135-.017-.33-.107-.312-.25-.008-.169.191-.242.33-.24.098 0 .179.02.24.064.061.042.1.11.106.184h.32c-.017-.41-.407-.543-.662-.532zm-10.123.026v1.836h1.2v-.28h-.868v-.5h.799v-.279h-.799v-.498h.862v-.28zm2.692 0v1.832h.33v-1.832zm.988 0v1.836h.332v-1.252h.017l.862 1.252h.287v-1.836h-.32v1.25h-.018l-.861-1.25zm1.924 0 .646 1.836h.412l.643-1.836h-.363l-.477 1.443h-.018l-.48-1.443zm2.164 0v1.836h1.199v-.28h-.865v-.5h.797v-.279h-.797v-.498h.857v-.28zm3.398 0v.279h.569v1.557h.326v-1.557h.568v-.28zm1.92 0v1.836h1.2v-.28h-.866v-.5h.797v-.279h-.797v-.498h.86v-.28zm-14.955.004v1.832h.742c.504-.011.619-.283.625-.498a.426.426 0 0 0-.396-.447v-.018c.084-.003.335-.136.324-.395.015-.194-.129-.48-.592-.474zm16.762 0v1.832h.623c.521.004.894-.334.886-.916 0-.6-.36-.918-.876-.916zm-16.43.271h.334c.226-.004.296.12.299.244.001.137-.107.27-.307.262h-.326Zm16.762.02h.283c.523.002.565.402.566.625-.005.242-.06.63-.577.628h-.272zm-16.762.726h.363c.265 0 .338.195.334.287.002.103-.06.253-.341.248h-.356Z"/></svg>
```

**Comment by service-paradis** [comment]
@elliotwutingfeng Did you use the path that I provided? If so, we'll nee another @simple-icons/pr-reviewers to approve this on!

**Comment by elliotwutingfeng** [comment]
Yes, I did.

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
@elliotwutingfeng & @service-paradis, I'm seeing some small differences throughout ([example](https://github.com/simple-icons/simple-icons/assets/15157491/489b040c-2eda-489c-8b0b-3b14cc589e1a) from bottom left of "X", with yours in red), which are likely just a precision issue, and some more pronounced ones in the tagline as a result of removing the excess points ([example](https://github.com/simple-icons/simple-icons/assets/15157491/447c8346-c65a-40c4-8593-11b5f573f449))

**Comment by adamrusted** [comment]
@elliotwutingfeng - are you still working on this one?

**Comment by elliotwutingfeng** [comment]
@adamrusted sorry for the lack of updates, I'd like to leave it to others now. Someone please take over this submission!

**Comment by adamrusted** [comment]
No worries, I'll leave this open for a while - and if nobody picks it up I'll close the PR to make it obvious the issue is good for contribution by someone else.

File: icons/saxobank.svg
Comment (PratyushJayachandran) [inline icons/saxobank.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Saxo Bank</title><path d="M20.944 16.62a.444.444 0 0 1-.192.211.658.658 0 0 1-.32.07h-.274v-1.26h.283c.125 0 .23.023.313.069a.443.443 0 0 1 .19.209.83.83 0 0 1 .065.35.835.835 0 0 1-.065.351Zm-.013-1.158a.952.952 0 0 0-.47-.11h-.636v1.838h.623a.98.98 0 0 0 .48-.11.74.74 0 0 0 .305-.317c.071-.137.106-.301.106-.493 0-.191-.035-.355-.106-.492a.736.736 0 0 0-.302-.316Zm-2.56.947h.8v-.279h-.8v-.499h.861v-.278h-1.195v1.837h1.202v-.279h-.87v-.502h.001Zm-2.334-.778h.568v1.559h.33v-1.559h.568v-.278h-1.466v.278Zm-.634.683a.728.728 0 0 0-.174-.104 1.14 1.14 0 0 0-.2-.065l-.173-.044a1.102 1.102 0 0 1-.112-.033.491.491 0 0 1-.101-.049.225.225 0 0 1-.072-.071.204.204 0 0 1 .014-.222.262.262 0 0 1 .114-.086.46.46 0 0 1 .178-.032.41.41 0 0 1 .241.066.245.245 0 0 1 .106.182h.32a.476.476 0 0 0-.088-.274.56.56 0 0 0-.231-.188.82.82 0 0 0-.343-.068.849.849 0 0 0-.346.068.58.58 0 0 0-.24.19.471.471 0 0 0-.09.284c0 .132.045.237.132.317a.83.83 0 0 0 .359.177l.209.054c.065.016.123.035.172.057.05.02.088.047.116.078a.182.182 0 0 1 .042.12.22.22 0 0 1-.047.137.306.306 0 0 1-.13.092.511.511 0 0 1-.19.033.534.534 0 0 1-.188-.032.28.28 0 0 1-.195-.255h-.327a.57.57 0 0 0 .095.31c.06.085.141.15.246.194s.23.067.372.067a.936.936 0 0 0 .377-.069.499.499 0 0 0 .32-.475.46.46 0 0 0-.045-.208.479.479 0 0 0-.121-.151Zm-2.633.095h.8v-.279h-.8v-.499h.863v-.278h-1.196v1.837h1.202v-.279h-.87l.001-.502Zm-1.67.39h-.02l-.478-1.446h-.365l.647 1.837h.411l.648-1.837h-.366l-.477 1.445Zm-1.689-.194h-.016l-.864-1.253h-.299v1.837h.333v-1.251h.015l.866 1.251h.296v-1.837H9.41v1.253Zm-2.125.584h.332v-1.837h-.332v1.837Zm-2.335-.78h.8v-.279h-.8v-.499h.862v-.278H4.618v1.837H5.82v-.279h-.87l.001-.502Zm-1.294.433c-.054.047-.14.07-.261.07h-.359v-.538h.367c.069 0 .128.012.178.038a.274.274 0 0 1 .156.251.227.227 0 0 1-.08.18Zm-.62-1.215h.337c.098 0 .173.023.222.07.05.045.075.104.075.175 0 .054-.013.1-.04.14a.263.263 0 0 1-.109.09.37.37 0 0 1-.155.032h-.33v-.507Zm.834.67a.405.405 0 0 0-.188-.059v-.018a.474.474 0 0 0 .162-.07.368.368 0 0 0 .116-.13.395.395 0 0 0 .043-.191.457.457 0 0 0-.066-.244.451.451 0 0 0-.198-.17.781.781 0 0 0-.332-.062h-.703v1.837h.743c.14 0 .257-.022.35-.065a.475.475 0 0 0 .209-.18.483.483 0 0 0 .068-.256.47.47 0 0 0-.057-.238.411.411 0 0 0-.147-.153Zm3.864-5.89.788-1.92.79 1.92H7.735Zm1.321-3.534H8.022l-2.44 5.73H6.83l.495-1.2h2.4l.493 1.195.784-1.157-1.945-4.568Zm6.145 2.722 1.843-2.722h-1.336l-1.192 1.768-1.16-1.768h-1.384l1.87 2.743-2.031 2.987h1.339l1.377-2.02 1.342 2.02h1.374l-2.042-3.008ZM1.477 8.505c0-.445.411-.696.966-.696.602 0 .918.29 1.082.641l1.124-.417c-.302-.772-1.124-1.25-2.132-1.25-1.348 0-2.233.814-2.233 1.807 0 1.97 3.229 1.246 3.229 2.376 0 .424-.439.692-1.065.692-.683 0-1.124-.37-1.31-.936L0 11.085c.265.978 1.193 1.608 2.412 1.608 1.332 0 2.317-.752 2.317-1.784 0-2.041-3.252-1.376-3.252-2.405ZM21 11.61c-.987 0-1.756-.785-1.756-1.873S20.014 7.866 21 7.866c1.002 0 1.762.784 1.762 1.872 0 1.089-.76 1.873-1.762 1.873Zm0-4.827c-1.685 0-2.994 1.33-2.994 2.955s1.31 2.955 2.994 2.955c1.684 0 2.999-1.33 2.999-2.955s-1.315-2.955-3-2.955Z"/></svg>
```
Taking up the PR

File: icons/saxobank.svg
Comment (PratyushJayachandran) [inline icons/saxobank.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Saxo Bank</title><path d="M20.944 16.62a.444.444 0 0 1-.192.212.66.66 0 0 1-.32.07h-.274v-1.26h.284c.124 0 .228.02.312.068.084.047.151.12.19.208.047.11.07.23.066.35 0 .14-.022.26-.066.352Zm-.014-1.158a.952.952 0 0 0-.47-.11h-.634v1.838h.62a.98.98 0 0 0 .48-.11.74.74 0 0 0 .308-.316c.075-.154.11-.323.106-.494a1.058 1.058 0 0 0-.106-.492.736.736 0 0 0-.304-.316Zm-2.56.948h.8v-.28h-.8v-.5h.862v-.278h-1.196v1.838h1.202v-.28h-.868v-.5Zm-2.332-.78h.568v1.56h.33v-1.56h.568v-.278h-1.466v.28Zm-.636.684a.728.728 0 0 0-.172-.104 1.14 1.14 0 0 0-.2-.066l-.174-.042a1.102 1.102 0 0 1-.112-.034.492.492 0 0 1-.1-.048.224.224 0 0 1-.074-.072.204.204 0 0 1 .014-.222.262.262 0 0 1 .114-.086.46.46 0 0 1 .18-.032c.098 0 .18.022.24.066.061.041.1.108.106.182h.32a.476.476 0 0 0-.088-.274.56.56 0 0 0-.232-.188.82.82 0 0 0-.342-.068.848.848 0 0 0-.346.068.58.58 0 0 0-.24.19.472.472 0 0 0-.09.284c0 .132.044.238.132.316.088.08.208.14.36.178l.208.054c.059.014.116.033.172.056a.3.3 0 0 1 .116.08.18.18 0 0 1 .042.12.22.22 0 0 1-.048.136.306.306 0 0 1-.128.092.512.512 0 0 1-.19.032.534.534 0 0 1-.188-.03.28.28 0 0 1-.194-.254h-.33c.006.12.038.224.096.308.06.086.14.15.246.194s.23.066.372.066a.936.936 0 0 0 .376-.068.5.5 0 0 0 .32-.476.46.46 0 0 0-.044-.208.48.48 0 0 0-.12-.15h-.002Zm-2.632.096h.8v-.28h-.8v-.5h.862v-.278h-1.194v1.838h1.2v-.28h-.868v-.5Zm-1.67.388h-.02l-.478-1.446h-.364l.646 1.838h.412l.646-1.838h-.366l-.476 1.446Zm-1.688-.194h-.016l-.866-1.252h-.298v1.838h.332v-1.252h.016l.866 1.252h.296v-1.838H9.41v1.252h.002Zm-2.126.586h.332v-1.838h-.332v1.838Zm-2.334-.78h.8v-.28h-.8v-.5h.86v-.278H4.62v1.838h1.202v-.28h-.87v-.5Zm-1.294.432c-.054.046-.14.07-.262.07h-.36v-.54h.368c.068 0 .128.014.178.04a.274.274 0 0 1 .156.252.228.228 0 0 1-.08.18v-.002Zm-.62-1.214h.336c.1 0 .172.022.222.068.05.044.078.109.076.176a.25.25 0 0 1-.04.14.264.264 0 0 1-.11.09.37.37 0 0 1-.154.032h-.33v-.506Zm.832.67a.406.406 0 0 0-.188-.06v-.018a.474.474 0 0 0 .164-.07.368.368 0 0 0 .158-.322.458.458 0 0 0-.066-.244.452.452 0 0 0-.2-.17.78.78 0 0 0-.33-.06h-.704v1.836h.744c.14 0 .258-.022.35-.066a.474.474 0 0 0 .208-.18.484.484 0 0 0 .07-.254.47.47 0 0 0-.058-.24.412.412 0 0 0-.148-.152Zm3.866-5.892.786-1.92.79 1.92H7.736Zm1.32-3.532H8.022l-2.44 5.73H6.83l.494-1.2h2.4l.492 1.194.784-1.156-1.944-4.568Zm6.146 2.72 1.84-2.72h-1.334l-1.192 1.768-1.16-1.768h-1.384l1.87 2.742-2.032 2.988h1.34l1.376-2.022 1.342 2.02h1.376L15.2 9.598l.002-.004ZM1.476 8.504c0-.444.412-.694.966-.694.604 0 .92.288 1.084.64l1.122-.418c-.3-.772-1.122-1.248-2.132-1.248-1.346 0-2.232.814-2.232 1.806 0 1.97 3.228 1.246 3.228 2.376 0 .424-.438.692-1.064.692-.684 0-1.124-.37-1.31-.936L0 11.086c.266.978 1.192 1.606 2.412 1.606 1.332 0 2.316-.75 2.316-1.782 0-2.042-3.252-1.378-3.252-2.406ZM21 11.61c-.986 0-1.756-.784-1.756-1.872s.77-1.872 1.756-1.872c1.002 0 1.764.784 1.764 1.872S22.004 11.61 21 11.61Zm0-4.826c-1.684 0-2.992 1.33-2.992 2.954s1.308 2.956 2.992 2.956c1.686 0 3-1.33 3-2.956s-1.314-2.954-3-2.954Z"/></svg>
```
Fix icon issues

**Review by uncenter — CHANGES_REQUESTED** [review]
Hmm I'm seeing quite a few large differences, especially with the bottom text here:

CleanShot 2024-06-16 at 21 47 59@2x

I suspect we are using different source icons then. I'm comparing yours to the `Logo/Digital use/SAXO_be_invested__saxo_blue_logo_rgb.svg` file from their media kit zip.

**Review by adamrusted — CHANGES_REQUESTED** [review]
I'm seeing the same differences as @uncenter I'm afraid @PratyushJayachandran!
image

**Comment by adamrusted** [comment]
Closing this PR, given it's not had updates from the original creator or the maintainer that took it over for some time. Will un-link from the issue so we can work on the issue fresh, should another contributor want to.

**Pull request closed without merging**
